### PR TITLE
Adjust converting auth config to JSON

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: handlers/firebase/handler.go [hunk 1]
```diff
@@ -259,10 +259,8 @@ func (h *handler) sendWithCredsJSON(ctx context.Context, msg courier.MsgOut, res
 		return courier.ErrChannelConfig
 	}
 
-	credentialsFileJSON, ok := credentialsFile.(map[string]string)
-	if !ok {
-		return courier.ErrChannelConfig
-	}
+	var credentialsFileJSON map[string]string
+	jsonx.MustUnmarshal(jsonx.MustMarshal(credentialsFile), &credentialsFileJSON)
```
[the remark this answers is not in the document]
Comment (norkans7) [reply handlers/firebase/handler.go:263]: the type we load from the DB is of type `map[string]any` and the other approach was not working

[the remark this answers is not in the document]
Comment (norkans7) [reply handlers/firebase/handler.go:263]: More on the issue on https://go.dev/play/p/ghaYeP3llx

[the remark this answers is not in the document]
Comment (rowanseymour) [reply handlers/firebase/handler.go:263]: But why do we need it as `map[string]string` anyway? It looks like we just marshal it again to pass to `WithCredentialsJSON`

[the remark this answers is not in the document]
Comment (norkans7) [reply handlers/firebase/handler.go:263]: we read `credentialsFileJSON["project_id"]`

**Pull request merged**